md/single-sign-on-with-cas.md

```diff
@@ -218,9 +218,25 @@ We recommend that you use LDAP as your master source for information, synchroniz
 
 ## Single sign-on with CAS using the REST API
 
+CAS is a browser oriented protocol (based on http automatic redirection, cookies, forms, etc...), therefore, we have securized browser oriented resources only. This is why only a subset of pages are handle to be automatically SSO CAS verified but not all the web application.
+
+The default AuthenticationFilter that manages CAS authentication applies only to the following pages: 
+
+* /portal
+* /mobile/\*
+* /portal.js/\*
+* /apps/\*
+* /services/\*
+
+REST API are not some of them but if a http session already exists thanks to cookies, REST API can be used.
+
+The recommended way to authenticate on Bonita BPM portal to use the REST API is to use the CAS server REST API.  
+It allows to retrieve authentication tickets to authenticate to Bonita BPM Portal.
```

Comment:
"Authenticate to" or "authenticate on" ?  Pick one, and use it throughout the doc page